drivers/include/mtd_mapper.h

```diff
@@ -96,7 +96,7 @@ typedef struct {
 typedef struct {
     mtd_dev_t mtd;                  /**< MTD context                         */
     mtd_mapper_parent_t *parent;    /**< MTD mapper parent device            */
-    uint32_t offset;                /**< Offset address to start this region */
+    uint32_t offset;                /**< Sector offset to start of this region */
```

Comment:
Minor minor nitpick, can we rename this member to something else, to break compilation on applications assuming the old struct interpretation?

Reply:
Yes I thought about that too.
But I wanted to get some feedback before renaming everything :wink:

How about just calling it `sector` instead?

Reply:
> How about just calling it `sector` instead?

Ok! feel free to squash in immediately

Reply:
`first_sector` or `base_sector`?

Reply:
`sector` is unambiguous here, so let's keep it short.